doc/rvv-intrinsic-spec.adoc

```diff
@@ -0,0 +1,523 @@
+
+The explicit (non-overloaded) intrinsics embeds the control described under <<control-to-vector-programming-model>> in the function name. This scheme makes intrinsics program easier to read given that the execution states be explicitly specified in the code.
+
+The implicit (overloaded) intrinsics, on the contrary, hide the explicit specifiations for `vtype` control. The implicit (overloaded) intrinsics aim to provide a generic interface to let users put values of different SEW and LMUL as the input operand.
+
+This section covers the general naming rule of the two types of intrinsics accordingly. Then, this section also enumerates the exceptions and the rationale behind them in <<chapter-exception-naming>>.
+
+[[policy-naming-scheme]]
+=== Policy naming scheme
+
+With the default policy scheme mentioned under <<control-to-policy>>, each intrinsics provide corresponding variants for their available control of the policy behavior. The following list enumerates the control of the policy behavior and their corresponding suffix.
+
+* No suffix: Represents an unmasked (`vm=1`) vector operation with tail-agnostic (`vta=1`)
+* `_tu` suffix: Represents an unmasked (`vm=1`) vector operation with tail-undisturbed (`vta=0`)
+* `_m` suffix: Represents a masked (`vm=0`) vector operation with tail-agnostic (`vta=1`), mask-agnostic (`vma=1`)
+* `_tum` suffix: Represents a masked (`vm=0`) vector operation with tail-undisturbed (`vta=0`), mask-agnostic (`vma=1`)
+* `_mu` suffix: Represents a masked (`vm=0`) vector operation with tail-agnostic (`vta=1`), mask-undisturbed (`vma=0`)
+* `_tumu` suffix: Represents a masked (`vm=0`) vector operation with tail-undisturbed (`vta=0`), mask-undisturbed (`vma=0`)
+
+Using `vadd` with SEW=8 and LMUL=1 as an example, the variants are:
+
+```c
+// vm=1, vta=1
+vint32m1_t __riscv_vadd_vv_i32m1(vint32m1_t op1, vint32m1_t op2, size_t vl);
+// vm=1, vta=0
+vint32m1_t __riscv_vadd_vv_i32m1_tu(vint32m1_t maskedoff, vint32m1_t op1,
+                                    vint32m1_t op2, size_t vl);
+// vm=0, vta=1, vma=1
+vint32m1_t __riscv_vadd_vv_i32m1_m(vbool32_t mask, vint32m1_t op1,
+                                   vint32m1_t op2, size_t vl);
+// vm=0, vta=0, vma=1
+vint32m1_t __riscv_vadd_vv_i32m1_tum(vbool32_t mask, vint32m1_t maskedoff,
+                                     vint32m1_t op1, vint32m1_t op2, size_t vl);
+// vm=0, vta=1, vma=0
+vint32m1_t __riscv_vadd_vv_i32m1_mu(vbool32_t mask, vint32m1_t maskedoff,
+                                    vint32m1_t op1, vint32m1_t op2, size_t vl);
+// vm=0, vta=1, vma=1
+vint32m1_t __riscv_vadd_vv_i32m1_tumu(vbool32_t mask, vint32m1_t maskedoff,
+                                      vint32m1_t op1, vint32m1_t op2,
+                                      size_t vl);
+```
+
+NOTE: When policy is set to "agnostic", users may either get tail (inactive maskedoff) element of undisturbed or all ones. Under this policy users should not assume the values within to be deterministic.
```

Comment:
This is not a strong enough statement. There's no guarantee you'll get all ones and there is no context to know what "tail" elements you'll get. There is no passthru operand. The compiler can pick any register.